msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -414,85 +544,105 @@ This section provides several code snippets for common scenarios that are relate
           return "Could not retrieve profile information from Microsoft Graph.";
         }
       }
-    }    
-    ```
+     }    
+
+     ```
 
-    For more information about how to use this class in command bot, see [Add single sign-on to Teams app](add-single-sign-on.md). And there's also a [command-bot-with-sso](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/command-bot-with-sso) sample project, which you can try sso command bot.
+    For more information about how to use class in command bot, see [add single sign-on to Teams app](add-single-sign-on.md). And there's a [command-bot-with-sso](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/command-bot-with-sso) sample project, which you can try SSO command bot.
```

Comment:
For more information about how to use class in command bot
->
For more information about how to implement SSO command handler in command bot, ...